stripe/src/main/java/com/stripe/android/model/BankAccount.java

```diff
@@ -44,8 +44,89 @@
     @Nullable private String mLast4;
     @Nullable private String mRoutingNumber;
 
+    public static Builder newBuilder(
+            @NonNull String accountNumber,
+            @Nullable String routingNumber,
```

Comment:
Should this (maybe?) omit the `routingNumber` since it's optional?

Reply:
@danj-stripe - I left this in, so we could maintain backwards compatibility for people already using this particular signature to make cards (which is likely many people).

Reply:
I don't have a well developed opinion, and I think either works fine.

It *definitely* made sense to me to leave `routingNumber` in the existing constructor to maintain backwards compatibility, where you just changed it from `@NonNull` to `@Nullable`.

My first thought was that since they'd be making code changes anyway to use the new `Builder`, it'd make sense to omit `routingNumber` from the `newBuilder()` method. This is an opportunity to clean up the signature to only include the required fields, since it's a new method/class and a new way to construct `BankAccount` objects.